src/System.Collections.Immutable/src/Properties/SecurityTransparent.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Security;
+
+[assembly: SecurityTransparent]
```

Comment:
Actually thinking a little more I kind of recall @jkotas suggesting we remove all security annotations for our libraries so perhaps he has an opinion on whether we should be marking this transparent or not.